subprojects/building-java-libraries/contents/index.adoc

```diff
@@ -126,12 +130,19 @@ include::{samples-dir}/groovy-dsl-files/build.gradle[tags=init]
 ----
 include::{samples-dir}/kotlin-dsl-files/build.gradle.kts[tags=init]
 ----
-<1> Public Bintray Artifactory repository
-<2> This is an example of a dependency which is exported to consumers, that is to say found on their compile classpath.
-<3> This is an example of a dependency which is used internally, and not exposed to consumers on their own compile classpath.
-<4> JUnit testing library
+<1> Use the `java-library` plugin
+<2> Public Bintray Artifactory repository, fore resolving project dependencies
+<3> This is an example of a dependency which is exported to consumers, that is to say found on their compile classpath.
+<4> This is an example of a dependency which is used internally, and not exposed to consumers on their own compile classpath.
+<5> JUnit testing library
+
+[NOTE]
+====
+Version numbers for the dependencies may be different for different Gradle versions.
+====
```

Comment:
Can you clarify? You mean the version numbers for the generated script right?

Reply:
Yes, that's right.

> Version numbers for the dependency declarations may vary for different Gradle versions.

Is the above any better?

Reply:
+1